### PR TITLE
fix note for lb in deploy-on-azure-aks.md

### PR DESCRIPTION
**Pull request by yiduoyunQ**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)
close issues [1850](https://github.com/pingcap/docs-tidb-operator/issues/1850)

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.3 (TiDB Operator 1.3 versions)
- [x] v1.2 (TiDB Operator 1.2 versions)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/deploy-on-azure-aks.md [hunk 1]
```diff
@@ -241,7 +241,12 @@ curl -O https://raw.githubusercontent.com/pingcap/tidb-operator/master/examples/
 
 > **注意：**
 >
-> 默认情况下，`tidb-cluster.yaml` 文件中 TiDB 服务的 LoadBalancer 配置为 "internal"。这意味着 LoadBalancer 只能在集群虚拟网络内部访问，而不能在外部访问。要通过 MySQL 协议访问 TiDB，您需要使用一个堡垒机进入集群节点或使用 `kubectl port-forward`。如果您想在互联网上公开访问 TiDB，并且知晓这样做的风险，您可以在 `tidb-cluster.yaml` 文件中将 LoadBalancer 的 "internal" 删除，默认创建的 LoadBalancer 将能够在外部访问。
+> 默认情况下，`tidb-cluster.yaml` 文件中 TiDB 服务的 LoadBalancer 配置为 "internal"。这意味着 LoadBalancer 只能在集群虚拟网络内部访问，而不能在外部访问。要通过 MySQL 协议访问 TiDB，您需要使用一个堡垒机进入集群节点或使用 `kubectl port-forward`。如果您想在互联网上公开访问 TiDB，并且知晓这样做的风险，您可以在 tidb-cluster.yaml 文件中将以下注释删除：
```
Comment (ran-huang) [inline zh/deploy-on-azure-aks.md:244]:
```suggestion
> 默认情况下，`tidb-cluster.yaml` 文件中 TiDB 服务的 LoadBalancer 配置为 "internal"。这意味着 LoadBalancer 只能在集群虚拟网络内部访问，而不能在外部访问。要通过 MySQL 协议访问 TiDB，您需要使用一个堡垒机进入集群节点或使用 `kubectl port-forward`。如果您想在互联网上公开访问 TiDB，并且知晓这样做的风险，您可以在 `tidb-cluster.yaml` 文件中将以下注释删除：
```

File: zh/deploy-on-azure-aks.md [hunk 2]
```diff
@@ -241,7 +241,12 @@ curl -O https://raw.githubusercontent.com/pingcap/tidb-operator/master/examples/
...
+> 
+> annotations:
+> service.beta.kubernetes.io/azure-load-balancer-internal: "true"
```
Comment (ran-huang) [inline zh/deploy-on-azure-aks.md:247]:
```suggestion
> ```yaml
> annotations:
>   service.beta.kubernetes.io/azure-load-balancer-internal: "true"
> ```
```

File: zh/deploy-on-azure-aks.md [hunk 3]
```diff
@@ -241,7 +241,12 @@ curl -O https://raw.githubusercontent.com/pingcap/tidb-operator/master/examples/
...
+> 
+> 之后重新创建的 LoadBalancer 及其关联的 TiDB 服务将能够在外部访问。
```
Comment (ran-huang) [inline zh/deploy-on-azure-aks.md:249]:
```suggestion
> 删除后，重新创建的 LoadBalancer 及其关联的 TiDB 服务将能够在外部访问。
```

File: en/deploy-on-azure-aks.md [hunk 4]
```diff
@@ -245,7 +245,12 @@ Refer to [configure the TiDB cluster](configure-a-tidb-cluster.md) to further cu
 
 > **Note:**
 >
-> By default, TiDB LoadBalancer in `tidb-cluster.yaml` is set to "internal", indicating that the LoadBalancer is only accessible within the cluster virtual network, not externally. To access TiDB over the MySQL protocol, you need to use a bastion to access the internal host of the cluster or use `kubectl port-forward`. You can delete the "internal" schema in the `tidb-cluster.yaml` file to expose the LoadBalancer publicly by default. However, notice that this practice may expose TiDB to risks.
+> By default, TiDB LoadBalancer in `tidb-cluster.yaml` is set to "internal", indicating that the LoadBalancer is only accessible within the cluster virtual network, not externally. To access TiDB over the MySQL protocol, you need to use a bastion to access the internal host of the cluster or use `kubectl port-forward`. You can delete the annotation following in the tidb-cluster.yaml file:
+> 
+> annotations:
+> service.beta.kubernetes.io/azure-load-balancer-internal: "true"
+> 
+> to expose the LoadBalancer publicly. However, notice that this practice may expose TiDB to risks.
```
Comment (ran-huang) [inline en/deploy-on-azure-aks.md:253]:
```suggestion
> By default, TiDB LoadBalancer in `tidb-cluster.yaml` is set to "internal", indicating that the LoadBalancer is only accessible within the cluster virtual network, not externally. To access TiDB over the MySQL protocol, you need to use a bastion to access the internal host of the cluster or use `kubectl port-forward`. If you understand the risks of exposing the LoadBalancer publicly, you can delete the following annotation in the `tidb-cluster.yaml` file:
> 
> ```yaml
> annotations:
> service.beta.kubernetes.io/azure-load-balancer-internal: "true"
> ```
>
> After deleting the annotation, the recreated LoadBalancer and its associated TiDB services will be externally accessible.
```

**Review by ran-huang — APPROVED** [review]

**Comment by ran-huang** [comment]
@KanShiori PTAL

**Review by csuzhangxc — APPROVED** [review]

**Pull request merged**